apps/chronicles/screens/components/SwitchScreen.tsx

```diff
@@ -0,0 +1,101 @@
+
+    const handleSwitchChange1 = (newState: boolean) => {
+        setIsActiveSwitch1(newState);
+    };
+
+    const handleSwitchChange2 = (newState: boolean) => {
+        setIsActiveSwitch2(newState);
+    };
+
+    const handleSmallSwitchChange = (newState: boolean) => {
+        setIsActiveSwitchSmall(newState);
+    };
+
+    const handleDisabledActiveState = (newState: boolean) => {
+        setIsDisabled(newState);
+        setDisabledActiveSwitch(newState);
+    };
+
+    return (
+        <ScrollView
+            contentInsetAdjustmentBehavior="automatic"
+            contentContainerStyle={styles.container}
+        >
+            <Typography variant="h2">Variations</Typography>
+            <Typography>Select between different states:</Typography>
+            <View style={styles.SwitchRow}>
+                <View style={styles.switchDescription}>
+                    <Switch active={isActiveSwitch1} onChange={handleSwitchChange1} />
+                    <Typography>Default Off</Typography>
+                </View>
+                <View style={styles.switchDescription}>
+                    <Switch active={isActiveSwitch2} onChange={handleSwitchChange2} />
+                    <Typography>Default On</Typography>
+                </View>
+            </View>
+            <Spacer />
+
+            <Typography variant="h2">Disabled</Typography>
+            <Typography>A switch can have multiple states:</Typography>
+            <View style={styles.SwitchRow}>
+                <View style={styles.switchDescription}>
+                    <Switch disabled={isDisabled} />
+                    <Typography>{"You can't turn me on"}</Typography>
+                </View>
+                <View style={styles.switchDescription}>
+                    <Switch
+                        active={isDisabledActiveSwitch}
+                        disabled={isDisabled}
+                        onChange={handleDisabledActiveState}
+                    />
+                    <Typography>{"You can't turn me off"}</Typography>
+                </View>
+            </View>
+            <Spacer />
+            <Typography variant="h2">Small</Typography>
+            <View style={styles.SwitchRow}>
+                <View style={styles.switchDescription}>
+                    <Switch.Small
+                        active={isActiveSwitchSmall}
+                        onChange={handleSmallSwitchChange}
+                        color="primary"
+                    />
+                    <Typography>
+                        {"This is a small switch, with all the same functionality"}
+                    </Typography>
+                </View>
+            </View>
+        </ScrollView>
+    );
+};
+
+const themeStyles = EDSStyleSheet.create(theme => ({
+    container: {
+        paddingHorizontal: theme.spacing.container.paddingHorizontal,
+        paddingVertical: theme.spacing.container.paddingVertical,
+    },
+    SwitchRow: {
+        paddingVertical: 8,
```

Comment:
Instead of hard coding this value, if you want spacing between your components, try to use the `<Spacer />` component in mad components. It comes with some predefined values

Reply:
this was taken from the button component, should i change it there aswell?